dissect/target/plugins/apps/webservers/webservers.py

```diff
@@ -0,0 +1,56 @@
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+WebserverRecord = TargetRecordDescriptor(
+    "application/log/webserver",
+    [
+        ("datetime", "ts"),
+        ("string", "remote_user"),
+        ("net.ipaddress", "ipaddr"),
+        ("uri", "file_path"),
+        ("wstring", "url"),
+        ("varint", "statuscode"),
+        ("varint", "bytessent"),
+        ("wstring", "referer"),
+        ("string", "useragent"),
+    ],
+)
+
+
+class WebserverPlugin(Plugin):
+
+    __namespace__ = "webserver"
+    TOOLS = [
+        "apache",
+        "nginx",
+        "iis",
+        "caddy",
+    ]
+
+    def __init__(self, target):
+        super().__init__(target)
+        self._plugins = []
+        for entry in self.TOOLS:
+            try:
+                self._plugins.append(getattr(self.target, entry))
+            except Exception:  # noqa
+                target.log.exception(f"Failed to load tool plugin: {entry}")
+
+    def check_compatible(self):
+        if not len(self._plugins):
+            raise UnsupportedPluginError("No compatible tool plugins found")
+
+    def _func(self, f):
+        for p in self._plugins:
+            try:
+                for entry in getattr(p, f)():
+                    yield entry
+            except Exception:
+                self.target.log.exception(f"Failed to execute tool plugin: {p._name}{f}")
+
+    @export(record=WebserverRecord)
+    def history(self):
```

Comment:
Would `access` not be more accurate?

Reply:
I agree, `logs` or `access` would be better. Which do you prefer? I think `iis` uses `logs`.

Reply:
I think `access` would be best for these currently. `logs` could be added later to be a sort of shorthand that calls the other log plugins (e.g. error).

Reply:
Added in [3106dde](https://github.com/fox-it/dissect.target/pull/127/commits/3106dde91dc2b556bb867dde4cee456e9fa374c3)